.storybook/preview.js

```diff
@@ -0,0 +1,19 @@
+import { ThemeProvider } from '@material-ui/core/styles'
+import theme from 'src/utils/theme'
+export const parameters = {
```

Comment:
What are these?

Reply:
Parameters are a set of static, named metadata about a story, used to control the behavior of Storybook features and addons.  This is boiler plate to enable the "actions" addon, which records handlers being invoked